backend/inspirehep/legacy/views.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import flask
+from flask import Blueprint, abort, request
+from sqlalchemy.orm.exc import NoResultFound
+
+from inspirehep.pidstore.api import PidStoreBase
+from inspirehep.records.utils import get_pid_for_pid
+
+blueprint = Blueprint("inspirehep_legacy", __name__, url_prefix="")
+
+
+@blueprint.route("/record/<path:recid>", methods=("GET",))
```

Comment:
isn't it ``int``?

Reply:
But we need to pass it as string for the query.